### PR TITLE
Issue51

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
This was part 2 of issue 51 :p

### CODE REVIEW

**Comment by nik-redhat** [comment]
Safe to mark as `draft` :joy:

**Comment by srijan-sivakumar** [comment]
Yup..I forgot to do that :p

File: redant_libs/redant_mixin.py [hunk 1]
```diff
@@ -1,6 +1,15 @@
+"""
+    Module Name:
```
Comment (nik-redhat) [inline redant_libs/redant_mixin.py:2]: Is this needed, or did you miss to add text for it?

Comment (srijan-sivakumar) [reply redant_libs/redant_mixin.py:2]: I kept it for future...Need to think on what can be part of the module doc string..

Comment (nik-redhat) [reply redant_libs/redant_mixin.py:2]: Ack.

**Pull request merged**